utils/FileHandler.cpp

```diff
@@ -0,0 +1,291 @@
+#include "FileHandler.hpp"
+
+namespace jbrush {
+
+Vector<Shape*> FileHandler::loadShapesFromDirectory(const QString& directory)
```

Comment:
Can we revert the name of this function? A directory is a folder, while a file is contained in a directory. We aren't loading shapes from a folder.